packages/libs/error-stack/src/lib.rs

```diff
@@ -124,6 +124,7 @@
 //! Could not parse configuration file
 //!              at main.rs:17:10
 //!       - Could not read file "config.json"
+//!       - 1 additional attachment
```

Comment:
```suggestion
//!       - 1 additional opaque attachment
```

Would perhaps be better?